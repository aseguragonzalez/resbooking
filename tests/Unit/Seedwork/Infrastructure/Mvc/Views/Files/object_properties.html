<section>
    <h1>Object properties</h1>
    <ul>
        <li>name: {{model->name}}</li>
        <li>age: {{model->age}}</li>
        <li>height: {{model->height}}</li>
    </ul>
    <hr>
    <address>
        <ol>
            <li>{{model->address->street}}, {{model->address->number}}</li>
            <li>{{model->address->city}}, {{model->address->postalCode}}</li>
            <li>{{model->address->state}}</li>
            <li>Updated at: {{model->address->updatedAt}}</li>
        </ol>
    </address>
</section>
