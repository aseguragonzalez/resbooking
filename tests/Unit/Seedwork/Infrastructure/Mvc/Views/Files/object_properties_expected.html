<section>
    <h1>Object properties</h1>
    <ul>
        <li>name: Freddy Krueger</li>
        <li>age: 45</li>
        <li>height: 1.75</li>
    </ul>
    <hr>
    <address>
        <ol>
            <li>Elm Street, 123</li>
            <li>Springwood, 12345-678</li>
            <li>Ohio</li>
            <li>Updated at: +2025-01-03T12:01:02+00:00</li>
        </ol>
    </address>
</section>
